### ISSUE TITLE
[Bug]: successful Form submission message is not assertive in Field and Input accessibility scenarios

### ISSUE THREAD

**Issue by adamsamec** [microsoft/fluentui#28381]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

yes

### Repro steps

1. Turn on screen reader.
2. Navigate to the [Field](https://fluentuipr.z22.web.core.windows.net/pull/25071/public-docsite-v9/storybook/iframe.html?id=concepts-developer-accessibility-stories--ticket-order-form-fields&viewMode=story) and [Input](https://fluentuipr.z22.web.core.windows.net/pull/25071/public-docsite-v9/storybook/iframe.html?id=concepts-developer-accessibility-stories--registration-form-inputs&viewMode=story) scenarios.
3. Fill in the forms so that they contain no errors.
4. Press the "Order ticket" or "Register" button, respectively.
5. Press the same submit button again if the first press does not work.

## Actual Behavior

Once the successful form submission message is shown, it is polite(does not interrupt the current announcement of the screen reader). This happens despite the aria-live="assertive" attribute being used on the message element. It has been tested to behave like that with JAWS and NVDA on Google Chrome and Microsoft Edge.

## Expected Behavior

Once the successful form submission message is shown, it should be assertive (should interrupt the current announcement of the screen reader).

### Requested priority

Normal

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by behowell** [comment]
Hi @adamsamec, this doesn't appear to be a bug with the controls in the FluentUI library, but rather a bug in the scenario pages in your PR https://github.com/microsoft/fluentui/pull/25071.

Is this issue intended to track a fix in your PR?
